### PR TITLE
Pass through context to serializers for included objects

### PR DESCRIPTION
**Pull request by zephyr-dev**

### CODE REVIEW

File: lib/jsonapi-serializers/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module JSONAPI
   module Serializer
-    VERSION = '0.6.1'
```
Comment (fotinakis) [inline lib/jsonapi-serializers/version.rb]: Please remove versioning changes, I handle this and our release procedures.

**Comment by zephyr-dev** [comment]
removed the version change

**Comment by fotinakis** [comment]
Thanks @zephyr-dev — some things have shifted in master recently, can you resolve those conflicts and I'll take another look?

File: spec/support/serializers.rb [hunk 2]
```diff
@@ -1,4 +1,20 @@
 module MyApp
+  class PostWithContext
```
Comment (fotinakis) [inline spec/support/serializers.rb]: Please move these down to the end of the module, so things continue to be ordered a bit more logically.

**Comment by fotinakis** [comment]
@zephyr-dev — looking forward to finishing this out with you, I think this will be great to fix the current context handling, and I can do some documentation after we get this in.

Let me know when you'll have a chance to resume this—just a tiny style thing above, and resolve the merge conflicts.

**Comment by fotinakis** [comment]
This has been finished in https://github.com/fotinakis/jsonapi-serializers/commit/f83d34f15b68a33e54f77a71d62a77cc4b1238a2 and released in gem v0.8.0.

**Pull request closed without merging**